iso.bbx

```diff
@@ -316,18 +409,26 @@
     {\printfield{howpublished}}%
 }
 
-%publication info
+% PUBLICATION INFO MACROS
+
+% By default only year is used, but for some entries,
+% we need full date (day or even time precision),
+% (e.g. patents, TV shows, newspapers, online news)
 
-\newbibmacro{fulldate}{%
-  \printtext{\csname mkbibrangeiso8601\endcsname{\thefield{date}}}%
+% Macro for printing full date (full = YYYY-MM-DD)
+\newbibmacro*{fulldate}{%
+  \printtext{\csname mkdaterangeiso\endcsname{}}%
```

Comment:
Since the macro name no longer contains non-letters the `\csname ...\endcsname` is not needed any more. I think the `\printtext` was never needed, so I'd try 
```suggestion
  \mkdaterangeiso{}%
```

Reply:
By trial and error this outputs the string `mkdaterangeiso`, but
```
\csname mkdaterangeiso\endcsname{}%
```
did the trick. I'll commit in a second.

Reply:
I don't see why
```latex
\csname mkdaterangeiso\endcsname
```
should behave different than
```latex
\mkdaterangeiso
```
but OK.

Reply:
Ok, I mistyped probably, it works! Fixed.